lib/modules/apostrophe-schemas/index.js

```diff
@@ -1402,6 +1409,15 @@ module.exports = {
       converters: {
         string: function(req, data, name, object, field, callback) {
           object[name] = self.apos.launder.integer(data[name], field.def, field.min, field.max);
+          if (field.required && !data[name]) {
+            return callback('required');
+          }
+          // This makes it possible to have a field that is not required, but min / max defined
+          // This allows the form to be saved and sets the value to null if no value was given by
+          // the user.
+          if (!data[name] && data[name] !== 0) {
```

Comment:
Looks good

Reply:
Hmm, same thought: can't you simplify this by looking at object[name] which contains the output of launder, guaranteed to be an integer or whatever field.def is?